include/macros.h

```diff
@@ -214,6 +214,13 @@
 #define LANGUAGE_ARRAY(jpn, eng, ger, fra) { eng, ger, fra }
 #endif
 
+// Macro for constants that change in 50 Hz N64 PAL versions.
+#if !OOT_PAL_N64
+#define FRAME_RATE_DEPENDENT(value60Hz, value50Hz) (value60Hz)
+#else
+#define FRAME_RATE_DEPENDENT(value60Hz, value50Hz) (value50Hz)
+#endif
```

Comment:
I like it better with the changes, thanks.

As for a name, I feel like its ok to make it shorter at least.. `FRAME_RATE_DEP()` ?
If we want to use "update rate" like other parts of the codebase, `UPD_RATE_DEP()`? 
idk, obv not ideal but I think shortening a bit is warranted. more ideas welcome

Reply:
hm I don't "update rate" because I'd like to distinguish between "frame rate" (60hz vs 50hz) and "update rate" (file select / pause menu / normal gameplay). But maybe those names are still too close. There's "VI rate" which is more obscure but is shorter at least

Reply:
What about UPDATE_CONST? Or will this macro be needed for non-constant expressions?

Reply:
should only be used by constants so I like `CONST`, I think the `UPDATE` part is too ambiguous though

Reply:
`FRAMERATE_CONST()`
`HZ_CONST()` for max shortness but prob too confusing lol

Reply:
Alright, I've gone with FRAMERATE_CONST